include/tvm/relay/attrs/nn.h

```diff
@@ -401,13 +401,18 @@ struct UpSamplingAttrs : public tvm::AttrsNode<UpSamplingAttrs> {
 struct PadAttrs : public tvm::AttrsNode<PadAttrs> {
   double pad_value;
   Array<Array<IndexExpr> > pad_width;
+  std::string pad_mode;
```

Comment:
Can we add test cases for different modes?

Reply:
I've added tests, please review again